protostar/cli/common_arguments.py

```diff
@@ -153,3 +153,17 @@
     description="Used by whitelisted users for deploying contracts in Alpha MainNet.",
     type="str",
 )
+
+CAIRO_PATH = ProtostarArgument(
+    name="cairo-path",
```

Comment:
Build cairo1 should use `linked-libraries` name

Reply:
#1604

Reply:
Don't need to change it here, just mentioning

Reply:
ok since there's a task for this, I think we can resolve